### PR TITLE
Update docs around benchmarks and plotting

### PR DESCRIPTION
**Pull request by tanertopal**
Especially remove old invalid docs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,63 +29,13 @@ $ pip install -e .[dev,cpu]
...
-   ```shell
-   $ ssh -i ~/.ssh/autofl_job.pem.xain ubuntu@18.185.67.166
-   ```
-
-3. Follow the logs of the container to see how things are running:
-
-```shell
-   $ docker logs -f $(docker ps -q)
-   ```
+To run training sessions, checkout the [benchmark package](https://github.com/xainag/xain/tree/master/xain/benchmark)
```
Comment (danieljanes) [inline README.md]: [...], see the [...]

File: xain/benchmark/README.md [hunk 2]
```diff
@@ -1,18 +1,76 @@
...
+In `~/.aws`, place two config files: `config` and `credentials`. Then execute the command:
+
+```shell
+export AWS_PROFILE=xain-xain
+```
 
-**Exception:**<br>
-The plot which compares the top performances of unitary and federated learning on the IID to Non-IID datasets aggregates top accuracies of a given benchmark group in the results directory and plots them.
-To do this you have to first run a group of benchmarks with e.g. the `train_remote_iid_noniid.sh` script in scripts. Afterwards you can plot the values using
+### Running a suit
```
Comment (danieljanes) [inline xain/benchmark/README.md]: Better: Running a benchmark suite

File: xain/benchmark/README.md [hunk 3]
```diff
@@ -1,18 +1,76 @@
...
-Alternatively when running `train_remote.sh` you can set it via the ENV variable `BENCHMARK_GROUP`
+You can see valid benchmark names by using
+
+```shell
+$ python -m xain.benchmark --helpfull
+```
+
+### Connect to running instances
+
+Start a benchmark suit
```
Comment (danieljanes) [inline xain/benchmark/README.md]: *suite

**Comment by tanertopal** [comment]
Applied requested changes

**Pull request merged**